dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/react';
+import { useEffect, useRef, useState } from 'react';
+import { getZIndex } from '../lib/getZIndex';
+import { useStickyElementHeight } from '../lib/useStickyElementHeight';
+import { useStickyElementMargin } from '../lib/useStickyElementMargin';
+import { useStickyObserver } from '../lib/useStickyObserver';
+import { StickyVideoButton } from './StickyVideoButton';
+
+const stickyStyles = (right: number) => css`
+	@keyframes fade-in-up {
```

Comment:
When testing this I found this animation hard to notice and the transition from playing inline to the content appearing at the bottom of the page a bit jarring. I'm no expert here but is there a way to make this transition more flowing?

Reply:
Agree, but I think we can probably abstract this from this PR to allow Anna and Marjan to get started on some of the other related tasks which are currently blocked. Does that sound reasonable?

I've created a Trello ticket for it [here](https://trello.com/c/39FjLF7J/254-sticky-video-fine-tune-animations)